docker/oso-monitor-sso/src/ops-run-in-loop

```diff
@@ -3,16 +3,15 @@
 # This is a temporary script until we get cron running as non-root (or find something else)
 
 if [ $# -lt 2 ] ; then
-    echo "Usage: $(basename $0) <sleep_time> <command>"
+    echo "Usage: $(basename $0) <sleep_time> <command> [arg] ..."
     exit 1
 fi
 
 SLEEP_TIME=$1
 shift
-COMMAND=$@
 
 # The purpose of this script is to run something in an infinite loop
 while true ; do
-    eval $COMMAND
+    "$@"
```

Comment:
eval is evil, so I re-wrote @twiest's script and updated it in oso-zagg-web. This change makes your container match that new version. See my commit 081f93e27d5427bd07fd0778d35213a9e319cd37 for the other change.